README.md

```diff
@@ -5,39 +5,57 @@
 
 This is a set of generators based on [yeoman-generator][yeoman-url] to aid in setting up commonly-used dependencies and application structure for One Design Company projects.
 
-The generator is made up of sub generators:
-  - `app` — Generates the app structure for one of the following:
-    - Craft 3
-    - Craft 2
-    - Static
-  - `git` — Initializes a git repo and sets up hooks and Github templates
-  - `styles` — Adds style dendencies like `suzy`, `one-sass-toolkit`, and other foundational styles
+## Generators
+
+There are many types of generators which can reference other generators. For example, a `craft` generator may compose a Craft project with `eslint`, `husky`, `style`, `scripts`, `gulp`, and `router`.
+
+### Platforms
+
+- `one-base:craft` - generates a basic Craft 2 or Craft 3 app
+- `one-base:static` - generates a simple front end app with no back end
+- `one-base:rails` - generates a Ruby on Rails app
+
+### Front End
+
+- `one-base:style` - installs common ODC style modules and dependencies and sets up a main.css
+- `one-base:scripts` - creates the ODC javascripts structure and sets up an example main.js
+- `one-base:router` - installs the one-router and sets up example templates
+- `one-base:gulp` - installs a gulp build configuration
+- `one-base:blendid` - installs a blendid build configuration
+
+### Development Tools
+
+- `one-base:git` - initializes a git repo and installs hooks and Github templates
+- `one-base:husky` - installs the javascript/JSON prettifier and ODC husky config
+- `one-base:eslint` - installs the ODC rules and rc files for auto-linting javascript
+- `one-base:rubocop` - installs the ODC rules for Ruby projects
+
 
 ## Installation
 
 ### Dependencies
 - [node.js][node-url] v6.11.1+
 - [npm][npm-url] v3.10.3+
 
-First, install [Yeoman][yeoman-url] and generator-one-base:
+First, install [Yeoman][yeoman-url]:
 
 ```bash
 npm install -g yo
```

Comment:
Should we recommend running this with `npx` instead of installing globally?

Reply:
npx, still needs things to be installed within a project, yo shouldn't care _where_ you run it.